playground/signer/playground_sign/delegate.py

```
@@ -176,6 +163,61 @@ def _get_online_input(
 
     return config
 
+def _collect_online_keys() -> list[SSlibKey]:
+    # TODO use value_proc argument to validate the input
+
+    while True:
+        click.echo(
+            f" 1. Sigstore\n"
+            f" 2. Google Cloud KMS\n"
+            f" 3. Azure Key Vault"
+        )
+        choice = click.prompt(
+            bold("Please select online key type"),
+            type=click.IntRange(0, 4),
+            default=0,
+            show_default=False,
```

What do you think about defaulting to sigstore (and not hiding the default)? 

I'm a bit conflicted:
* it's pretty clear the sigstore support is a little experimental still, but...
* mostly for demonstration purposes I'd like to have default values that (once the current dependency fiasco is solved) just work if you keep pressing enter

---

Yes, I agree. Default to sigstore makes sense as that would make it easy to get on board. I'll update. I think for a real situation, when you are actually configuring a trust root, you don't blindly press enter, you think carefully about your config so there is no need to rely on the default values.

---

Also one thing I would like to have feedback on. The local testing key, before it was based on a "known value", not it's a "hidden option" (4). You are ok with that?

---

LGTM

---

hiding option 4 seems good to me